dev/com.ibm.ws.cdi.weld/src/com/ibm/ws/cdi/impl/managedobject/CDIEJBManagedObjectFactoryImpl.java

```diff
@@ -189,9 +223,6 @@ private synchronized void doInitialization() {
 	        } else { 
```

Comment:
This `else` is wrongly indented (it has a tab at the start of the line instead of eight spaces)